pac4j-core/src/main/java/org/pac4j/core/util/JavaSerializationHelper.java

```diff
@@ -57,40 +64,96 @@ public String serializeToBase64(final Serializable o) {
     }
 
     /**
-     * Unserialize a base64 String into a Java object.
+     * @deprecated use {@link #deserializeFromBase64} instead
+     */
+    @Deprecated
+    public Serializable unserializeFromBase64(final String base64) {
+        return deserializeFromBase64(base64);
+    }
+
+    /**
+     * Deserialize a base64 String into a Java object.
      *
      * @param base64 the serialized object as a base64 String
-     * @return the unserialized Java object
+     * @return the deserialized Java object
      */
-    public Serializable unserializeFromBase64(final String base64) {
-        return unserializeFromBytes(Base64.getDecoder().decode(base64));
+    public Serializable deserializeFromBase64(final String base64) {
```

Comment:
I also realize we can't change the name in 3.5. Its breaks backward compatibility...